### ISSUE TITLE
Experimental support for audio conversion to opus

### ISSUE THREAD

**Issue by lact500** [Ajatt-Tools/PasteImagesAsWebP#34]
Hi.

Some time ago I added a review on the [**Addon's page**](https://ankiweb.net/shared/info/1151815987) about two features I thought would be rather useful. The response I got was that some features had, understandably, been stalled for a while. So I started messing around with the addon's files locally and, through trial and error, copy-paste, a bit of help from AI, and lots and lots of troubleshooting, the experimental implementation of these two features was made possible:

1. Add a list of extensions as exceptions for image conversion. Configurable from the Advanced Settings window.
2. Experimental implementation of opus conversion for audio files. Configurable from the Advanced Settings window.

I tried not to modify the original code too much, and tried to follow a similar logic to the existing image conversion for the new functionality. These features have been tested for a few days without issues and here's a few findings.

a. Audio conversion uses ffmpeg since it was already used.
b. Conversion works for cards whether they have an image, a sound file, or both.
c. Audio conversion can be turned off individually from the advanced settings window.
d. Audio conversion has been implemented and tested **only** for bulk conversion.
e. Bulk-reconvert affects both image and audio conversion simultaneously.

[Here's the Fork](https://github.com/lact500/PasteImagesAsWebP_Opus) with the changes I made. But, since I'm not actually a programmer, I'd rather leave this here so someone with more experience can look at these changes and determine if they should be added to the addon's code or not.

That's it for now.
Greetings.

**Comment by tatsumoto-ren** [comment]
Hi. Thanks a lot for the work you've done! I'll try to incorporate it into the addon.

**Comment by tatsumoto-ren** [comment]
Excluded image extensions can be made into a separate PR since large diffs are harder to review.

**Comment by lact500** [comment]
> Hi. Thanks a lot for the work you've done! I'll try to incorporate it into the addon.

> Excluded image extensions can be made into a separate PR since large diffs are harder to review.

Right. Since my media collection isn't that big, I hand't thought about that. Maybe creating a separate file to manage both audio and image exclusions would be better. Thank you for taking a look at the fork and I hope it helps the addon improve.

**Comment by tatsumoto-ren** [comment]
I've merged your changed in the latest release. The next step is to add support for the note-added, paste and drag-and-drop events.
